### PR TITLE
🌐 Add French translation for Project Generation

### PR DESCRIPTION
**Pull request by Smlep**
Hi everyone!

Here is the PR to translate the Project generation section into french.
See the french translation tracking issue [here](https://github.com/tiangolo/fastapi/issues/1972).

Thanks for the reviews👌!

### CODE REVIEW

File: docs/fr/docs/project-generation.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* Intégration Traefik, comprenant la génération automatique de certificat **HTTPS** Let's Encrypt.
+* GitLab **CI** (intégration continue), comprenant des tests pour le frontend et le backend.
+
+## Full Stack FastAPI Couchbase
+
+GitHub : <a href="https://github.com/tiangolo/full-stack-fastapi-couchbase" class="external-link" target="_blank">https://github.com/tiangolo/full-stack-fastapi-couchbase</a>
+
+⚠️ **ATTENTION** ⚠️
+
+Si vous démarrez un nouveau projet de zéro, allez voir les alternatives ici.
```
Comment (NastasiaSaby) [inline docs/fr/docs/project-generation.md:61]: A link is missing, isn't it?

File: docs/fr/docs/project-generation.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+GitHub : <a href="https://github.com/microsoft/cookiecutter-spacy-fastapi" class="external-link" target="_blank">https://github.com/microsoft/cookiecutter-spacy-fastapi</a>
+
+## Modèles d'apprentissage automatique avec spaCy et FastAPI - Fonctionnalités
+
+* Intégration d'un modèle NER **spaCy**.
+* Formatage de requête pour **Azure Cognitive Search**.
+* Serveur Python web **prêt à utiliser en production** utilisant Uvicorn et Gunicorn.
+* Déploiement CI/CD Kubernetes pour **Azure DevOps** (AKS).
+* **Multilangues**. Choisissez facilement l'un des languages intégrés à spaCy durant le mise en place du projet.
```
Comment (NastasiaSaby) [inline docs/fr/docs/project-generation.md:83]: Is it "langues" like a language we speek, or language like a programming language?

File: docs/fr/docs/project-generation.md (hunk 1 above)
Comment (Smlep) [reply docs/fr/docs/project-generation.md:61]: No it isn't, it's the same in English https://fastapi.tiangolo.com/project-generation/

But I agree that the formulation is really ambiguous (in english too), I'll change this

File: docs/fr/docs/project-generation.md (hunk 2 above)
Comment (Smlep) [reply docs/fr/docs/project-generation.md]: It was `langues` https://spacy.io/usage/models.
I'm fixing this

**Review by NastasiaSaby — APPROVED** [review]

**Review by krocdort — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks a lot @Smlep ! :man_dancing: :tada: 

And thank you as always for the reviews @NastasiaSaby and @krocdort :bow: :clap:

**Pull request merged**